shared/src/commonMain/kotlin/com/twilio/passkeys/extensions/StringExtensions.kt

```diff
@@ -27,5 +27,14 @@ import kotlin.io.encoding.ExperimentalEncodingApi
  */
 @OptIn(ExperimentalEncodingApi::class)
 internal fun String.b64Decode(): ByteArray {
-  return Base64.UrlSafe.decode(this)
+  return Base64.UrlSafe.decode(this.padBase64())
+}
+
+private const val BASE64_BLOCK_SIZE = 4
+private const val NO_PADDING_NEEDED = 0
+
+// Helper function to pad the Base64 string if needed
+private fun String.padBase64(): String {
```

Comment:
is it worthy to add unit testing for this?

Reply:
Added